### PR TITLE
[WIP] feat(patterns/iconkit): submit the latest iconkit

### PR DESCRIPTION
**Pull request by melanieanglardLMFR**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a breaking change?

read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)

- [x] Yes _It could introduce a BC but i supposed that icons are not already really used in project_
- [ ] No

## Describe the changes

As LMFR BU we propose our latest updated iconkit to be able to use MOZAIC icon kit as a reference in our project

Github issue number or Jira issue URL: N/A

### CODE REVIEW

File: src/icons/48px/User_Payment_Colors_AmericanExpress_48px.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...00.76H78l3.9,4.35,4-4.35H96c2.51,0,5.32.71,5.32,4.39s-2.73,4.42-5.51,4.42Zm19.09-.6a3.9,3.9,0,0,1,.52,2.44V114h-3.19v-1.65c0-.79.08-2-.52-2.59s-1.16-.6-2.33-.6h-3.37V114h-3.19V100.76h7.31a7.84,7.84,0,0,1,3.83.64,3,3,0,0,1,1.61,3,3.7,3.7,0,0,1-2.25,3.53,3.53,3.53,0,0,1,1.58,1.08Zm13-5.43h-7.43v2.4h7.24v2.7h-7.24v2.62H128V114H117.37V100.76H128Zm1.91,10.5v-2.85H136a1.77,1.77,0,0,0,1.28-.34,1.17,1.17,0,0,0,.08-1.64l-.08-.09a1.8,1.8,0,0,0-1.13-.3c-3-.11-6.67.12-6.67-4.08,0-1.95,1.2-4,4.53-4h6.34v2.85h-5.81a2.15,2.15,0,0,0-1.28.23,1.06,1.06,0,0,0-.45,1,1,1,0,0,0,.68,1,3.7,3.7,0,0,0,1.24.15h1.68c1.73,0,2.93.33,3.64,1.08a3.46,3.46,0,0,1,.94,2.78c0,2.89-1.76,4.2-5,4.2Zm22.8-1.65-.15.18-.22.23-.27.26-.11.08-.34.22h-.07a6.36,6.36,0,0,1-1.65.53h-.11l-.49.07h-7v-2.85h6.12a1.68,1.68,0,0,0,1.27-.34,1.08,1.08,0,0,0,.38-.86,1,1,0,0,0-.42-.79,1.66,1.66,0,0,0-1.08-.3c-3-.11-6.72.12-6.72-4.08a3.85,3.85,0,0,1,1-2.63A.71.71,0,0,1,143,102l.15-.15.15-.11.19-.15.15-.07.22-.15.15-.08.3-.11h.11l.34-.11h.11l.42-.08h7.5v2.85h-6.23a1.46,1.46,0,0,0-.86.22h0a1.11,1.11,0,0,0,.19,2,3.68,3.68,0,0,0,1.23.15h1.73a7,7,0,0,1,3,.6,3.14,3.14,0,0,1,.6.49s0,0,.07.08a3.6,3.6,0,0,1,.9,2.7,4.27,4.27,0,0,1-.71,2.4Z"/></g></svg>
```
Comment (gael-boyenval) [inline src/icons/48px/User_Payment_Colors_AmericanExpress_48px.svg]: not sure this one was flattened

Comment (gael-boyenval) [inline src/icons/48px/User_Payment_Colors_AmericanExpress_48px.svg]: no colors icons

File: src/icons/48px/User_Payment_Colors_CarteBleue_48px.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...k="http://www.w3.org/1999/xlink" viewBox="0 0 192 192"><defs><style>.cls-1{fill:none;}.cls-2{fill:url(#Dégradé_sans_nom_52);}.cls-3{fill:#fff;}</style><linearGradient id="Dégradé_sans_nom_52" x1="-155.92" y1="-2012.12" x2="-20.26" y2="-1875.92" gradientTransform="matrix(1.14, 0, 0, -0.79, 196.32, -1448.21)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#109a72"/><stop offset="0.3" stop-color="#008a8d"/><stop offset="0.69" stop-color="#01679b"/><stop offset="1" stop-color="#03376f"/></linearGradient></defs><title>User_Payment_Colors_CarteBleue_48px</title><g id="Square"><rect class="cls-1" width="192" height="192"/></g><g id="Card"><rect class="cls-2" x="16" y="40" width="160" height="112" rx="8"/><path class="cls-3" d="M71.12,94.12H98.36c-.24-5.36-1.64-10.92-5.8-14.48-4.92-4.24-13.6-5.8-21.36-5.8s-17,1.72-22,6.28C44.92,84,44,90.32,44,96s1.8,12.84,6.48,16.84,13,5.32,20.72,5.32,15.68-1.24,20.6-5.28S98.4,102.12,98.4,96H71.12Z"/><path class="cls-3" d="M100.16,96V117.2H138a10.48,10.48,0,0,0,10-10.48A10.76,10.76,0,0,0,138,96Z"/><path class="cls-3" d="M147.32,84.4a9.65,9.65,0,0,0-9.6-9.68h-.12a11.24,11.24,0,0,0-1.16,0H100.16V94.12h38.2A9.8,9.8,0,0,0,147.32,84.4Z"/></g></svg>
```
Comment (gael-boyenval) [inline src/icons/48px/User_Payment_Colors_CarteBleue_48px.svg]: no colors icons

File: src/icons/48px/User_Payment_Colors_Cheque_48px.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...ix(-1.14, 0, 0, 0.79, 2615.83, 1607.72)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fff"/><stop offset="0.53" stop-color="#f1fdfe"/><stop offset="1" stop-color="#ecfcfd"/></linearGradient></defs><title>User_Payment_Colors_Cheque_48px</title><g id="Square"><rect class="cls-1" width="48" height="48"/></g><g id="Icon"><rect class="cls-2" x="4" y="10" width="40" height="28" rx="2"/><path class="cls-3" d="M42,11a1,1,0,0,1,1,1V36a1,1,0,0,1-1,1H6a1,1,0,0,1-1-1V12a1,1,0,0,1,1-1H42m0-1H6a2,2,0,0,0-2,2V36a2,2,0,0,0,2,2H42a2,2,0,0,0,2-2V12a2,2,0,0,0-2-2Z"/><path class="cls-4" d="M37,20H24.41l2-2H37a1,1,0,0,0,0-2H28.41l.8-.79a1,1,0,0,0-1.42-1.42L25.59,16H11a1,1,0,0,0,0,2H23.59l-2,2H11a1,1,0,0,0,0,2h8.59l-.8.79a1,1,0,0,0,0,1.42,1,1,0,0,0,1.42,0L22.41,22H37a1,1,0,0,0,0-2Z"/><rect class="cls-3" x="10" y="27" width="12" height="7" rx="1"/><path class="cls-5" d="M25,34a1,1,0,0,1-.37-.07,1,1,0,0,1-.56-1.3C26.32,27,27.5,27,28,27s1.38,0,3.09,3.67C32.57,28,33,28,33.5,28s1.07,0,3.12,4H38a1,1,0,0,1,0,2H36a1,1,0,0,1-.89-.55c-.51-1-1.15-2.19-1.61-2.92-.46.73-1.1,1.9-1.61,2.92A1,1,0,0,1,31,34a1,1,0,0,1-.89-.63A23,23,0,0,0,28,29.25a23,23,0,0,0-2.07,4.12A1,1,0,0,1,25,34Zm8-4.16Z"/></g></svg>
```
Comment (gael-boyenval) [inline src/icons/48px/User_Payment_Colors_Cheque_48px.svg]: no colors icons

File: src/icons/48px/User_Payment_Colors_Illico_48px.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...3.72,3.72,0,0,1-2.76,3.56,3.66,3.66,0,0,1-5-3.24,7.35,7.35,0,0,1,.4-3.08,3.69,3.69,0,0,1,2.84-2.4l1.28-.16.6,0A2.55,2.55,0,0,1,124,97.76a6.92,6.92,0,0,1,.28.76A9.46,9.46,0,0,1,124.4,101.52Z"/><path class="cls-6" d="M144,97.08a6.63,6.63,0,0,0-5.76-3.6,8.36,8.36,0,0,0-3.56.64,7,7,0,0,0-4.12,5.16,8.78,8.78,0,0,0-.12,3.68,7.17,7.17,0,0,0,5.44,5.16,9.15,9.15,0,0,0,1.48,0,7.37,7.37,0,0,0,6.88-4l.15-.31a6.92,6.92,0,0,0,.49-3.88A8.85,8.85,0,0,0,144,97.08ZM142.2,102a3.85,3.85,0,0,1-1.72,2.68,4.7,4.7,0,0,1-2.84.92,4.56,4.56,0,0,1-4.72-3.4,6.06,6.06,0,0,1,0-2.68,4.05,4.05,0,0,1,2.8-3.12,7.6,7.6,0,0,1,2.32-.36,4,4,0,0,1,3.84,2.8A6.52,6.52,0,0,1,142.2,102Z"/><path class="cls-6" d="M111.08,100.76c0-1.08,0-2.44,0-3.24V95.16c0-.72-.24-1-1-1a1.89,1.89,0,0,0-1,.28.8.8,0,0,1-.88,0l-.64-.28a8.8,8.8,0,0,0-8.24,1,7.77,7.77,0,0,0-2.8,5.16,6.53,6.53,0,0,0,.16,2.64,7,7,0,0,0,5.2,4.92,8,8,0,0,0,6.2-1.32l.88-.6a7.4,7.4,0,0,0,.44,1,1.18,1.18,0,0,0,1.76.64,1.11,1.11,0,0,0,.76-1.48A35.68,35.68,0,0,1,111.08,100.76Zm-2.52.84a3,3,0,0,1-1,2.44,6.31,6.31,0,0,1-2.72,1.48,4.86,4.86,0,0,1-4.68-1.36,3.79,3.79,0,0,1-1-1.88,5,5,0,0,1,3.32-5.84,6.39,6.39,0,0,1,3.36-.12,2.87,2.87,0,0,1,2.6,2.68,8,8,0,0,1,.12,2Z"/></g></svg>
```
Comment (gael-boyenval) [inline src/icons/48px/User_Payment_Colors_Illico_48px.svg]: no colors icons

File: src/icons/48px/User_Payment_Colors_LM_48px.svg [hunk 5]
```diff
@@ -0,0 +1 @@
...ZM75.88,88c-1.36,1.36-.88,3,.76,4.72s3.32,2.12,4.68.72.92-3-.76-4.68S77.24,86.64,75.88,88Z"/><polygon points="85.56 81.76 83.96 76.88 86.48 74.32 88.64 82.68 92 86.04 89.8 88.24 86.44 84.88 78.48 82.36 80.88 79.92 85.48 81.72 85.56 81.76"/><polygon points="125.24 111.36 133.84 102.76 131.52 100.44 122.92 109.08 125.24 111.36"/><polygon points="92.44 78.48 99.12 75.28 94.2 80.2 96.16 82.2 104.76 73.6 102.04 70.84 95.56 74 95.56 74 98.8 67.6 96 64.76 87.4 73.4 89.4 75.4 94.32 70.44 91.12 77.16 92.44 78.48"/><polyline points="107.32 79.68 105.88 81.16 109.16 84.48 107.4 86.24 104.12 82.92 102.24 84.8 106.24 88.8 104.52 90.56 98.32 84.36 106.92 75.72 113.08 81.96 111.36 83.72 107.32 79.68"/><polygon points="115.44 101.56 121.28 107.44 123.24 105.48 119.6 101.8 126.24 95.12 124.04 92.96 115.44 101.56"/><polygon points="138.8 107.72 136.04 116.88 136.04 116.88 141.96 110.92 144 112.96 135.4 121.6 132.72 118.88 135.08 109.48 135.08 109.44 129.2 115.36 127.2 113.36 135.8 104.72 138.8 107.72"/><path d="M114,100.12l1.68-5.88h0a3.68,3.68,0,0,0,4.2-.84,3.53,3.53,0,0,0-.24-4.92h0L115,83.84l-8.56,8.64,2.16,2.2,7-7,1.68,1.68a1.56,1.56,0,1,1-2,2.36s-.11-.1-.16-.16L113.48,90l-1.8,7.72Z"/></g></svg>
```
Comment (gael-boyenval) [inline src/icons/48px/User_Payment_Colors_LM_48px.svg]: no colors icons

File: src/icons/48px/User_Payment_Colors_MasterCard_48px.svg [hunk 6]
```diff
@@ -0,0 +1 @@
...8,3.08,3.08,0,0,0-1.4-.28A3.75,3.75,0,0,0,132,124a3.51,3.51,0,0,0-2,2.08,4.15,4.15,0,0,0-.28,1.6,4,4,0,0,0,.28,1.56,4.25,4.25,0,0,0,.8,1.28,3.88,3.88,0,0,0,1.2.8,4,4,0,0,0,1.48.32,2.94,2.94,0,0,0,1.4-.32,2.34,2.34,0,0,0,1-.8v.92h1.64v-10.6Zm0,7.8a2.87,2.87,0,0,1-.48.76,2.33,2.33,0,0,1-.72.48,2.24,2.24,0,0,1-.92.2,2.43,2.43,0,0,1-1-.2,1.78,1.78,0,0,1-.72-.52,2.58,2.58,0,0,1-.6-1.68,3.45,3.45,0,0,1,.16-1A3.2,3.2,0,0,1,132,126a2.33,2.33,0,0,1,.72-.52,2.43,2.43,0,0,1,1-.2,2.24,2.24,0,0,1,.92.2,1.78,1.78,0,0,1,.72.52,2.09,2.09,0,0,1,.48.76,3.45,3.45,0,0,1,.16,1A3.86,3.86,0,0,1,135.76,128.68Z"/><path class="cls-6" d="M109.24,126a2.33,2.33,0,0,1,.72-.48,2.43,2.43,0,0,1,1-.2h0a2.66,2.66,0,0,1,1,.2,1.51,1.51,0,0,1,.8.48l.8-1.32a3.91,3.91,0,0,0-1-.6,4.15,4.15,0,0,0-1.6-.28,4.36,4.36,0,0,0-1.64.28,3.9,3.9,0,0,0-1.28.84,3.73,3.73,0,0,0-.84,1.24,4.19,4.19,0,0,0-.32,1.6,4,4,0,0,0,.32,1.56,3.73,3.73,0,0,0,.84,1.24,3.9,3.9,0,0,0,1.28.84,4.39,4.39,0,0,0,1.64.32,4.19,4.19,0,0,0,1.6-.32,5.45,5.45,0,0,0,1-.56l-.8-1.32a2.25,2.25,0,0,1-.8.48,3.8,3.8,0,0,1-1,.16h0a3.45,3.45,0,0,1-1-.16,2.33,2.33,0,0,1-.72-.52,2,2,0,0,1-.52-.76,2.62,2.62,0,0,1-.16-1,2.83,2.83,0,0,1,.16-1A2,2,0,0,1,109.24,126Z"/></g></svg>
```
Comment (gael-boyenval) [inline src/icons/48px/User_Payment_Colors_MasterCard_48px.svg]: no colors icons

File: src/icons/48px/User_Payment_Colors_Oney_48px.svg [hunk 7]
```diff
@@ -0,0 +1 @@
...,0,1,111.62,82a3.91,3.91,0,0,0,.48,0ZM104,93.42h16.4c-.07-3-4.05-6.46-7.65-6.64A8.46,8.46,0,0,0,104,93.42Z"/><path class="cls-3" d="M55,82c.69.11,1.38.2,2.06.34A14.05,14.05,0,0,1,64.6,105.1a13.13,13.13,0,0,1-11.06,4.82c-6-.28-10.29-3.29-12.58-8.92a23,23,0,0,1-.85-3.55c0-.14,0-.28-.08-.42V94.84a3.38,3.38,0,0,0,.08-.34A13.74,13.74,0,0,1,50.8,82.34c.72-.16,1.46-.24,2.19-.36Zm8,14c-.08-.66-.12-1.31-.25-2a9,9,0,0,0-10.55-7.08,8.62,8.62,0,0,0-1.32.37,9.23,9.23,0,0,0-5.67,11,9,9,0,0,0,10,6.8A9.21,9.21,0,0,0,63,96Z"/><path class="cls-3" d="M132.06,118q2-4.86,4-9.72a.8.8,0,0,0,0-.53Q130.66,94.86,125.26,82l0,0h5.6a1.17,1.17,0,0,0,0,.2l7.77,19a2.54,2.54,0,0,0,.14.25l.2-.46c1-2.4,1.95-4.78,2.92-7.15q2.43-6,4.85-11.89H152c0,.15-.08.3-.13.45q-5.25,12.8-10.5,25.6l-4.08,10Z"/><path class="cls-3" d="M84.52,82c.7.11,1.41.18,2.1.35,4.44,1.14,7.58,3.73,8.77,8.28a14.32,14.32,0,0,1,.4,3.51c0,5.14,0,10.28,0,15.42v.36H90.74v-.52c0-4.9,0-9.81,0-14.71a14.85,14.85,0,0,0-.36-3.25,5.74,5.74,0,0,0-4.7-4.57,10.5,10.5,0,0,0-4.6,0,6,6,0,0,0-4.94,5.23,17.35,17.35,0,0,0-.21,2.7v15.13H70.88a2,2,0,0,1,0-.24c0-5.38,0-10.77,0-16.15A10.84,10.84,0,0,1,74.74,85a12.32,12.32,0,0,1,6.94-2.91c.16,0,.32-.06.47-.09Z"/></g></svg>
```
Comment (gael-boyenval) [inline src/icons/48px/User_Payment_Colors_Oney_48px.svg]: no colors icons

File: src/icons/48px/User_Payment_Colors_PayLib_48px.svg [hunk 8]
```diff
@@ -0,0 +1 @@
...1,1,0,0,1,.52-1,24.73,24.73,0,0,1,14-4ZM119.32,87a2.45,2.45,0,0,0,2,2.72,5.1,5.1,0,0,0,1.56.08,2.32,2.32,0,0,0,2.24-2.28V86.6a2.43,2.43,0,0,0-2-2.32,5.61,5.61,0,0,0-1.16,0A2.45,2.45,0,0,0,119.32,87Z"/><path class="cls-3" d="M81,84.24h4.88v.56c0,4.28,0,8.56,0,12.84a11.21,11.21,0,0,0,.12,1.64,1.79,1.79,0,0,0,1.28,1.44,7.27,7.27,0,0,0,4.52,0,1.91,1.91,0,0,0,1.4-1.84c0-.48.08-.92.08-1.4,0-4.2,0-8.44,0-12.64v-.56H98c0,.16,0,.32,0,.48,0,7.4,0,14.8,0,22.2a10.59,10.59,0,0,1-.24,2.24A3.57,3.57,0,0,1,94.84,112a30.53,30.53,0,0,1-5.12.24v-4h2.64a.79.79,0,0,0,.8-.76V104.8L89.8,105A12.44,12.44,0,0,1,85,104.4,5.63,5.63,0,0,1,81,99.2c0-.76,0-1.52,0-2.28,0-4,0-8.08,0-12.12Z"/><path class="cls-3" d="M78.28,84.2V105H73.44v-1a10.16,10.16,0,0,1-1.76.68,11.86,11.86,0,0,1-6.68-.16,6.2,6.2,0,0,1-4.4-5.08,16.45,16.45,0,0,1-.4-3.12,26.43,26.43,0,0,1,.36-6.88c.8-3.48,2.84-5.24,6.4-5.52a15.37,15.37,0,0,1,4.28.24,7.3,7.3,0,0,1,3,1.44V84.2ZM73.44,94.36h0c0-.56,0-1.12,0-1.68s0-1.12,0-1.68c-.12-1.8-1-2.84-2.6-3A17.31,17.31,0,0,0,68,88a2.73,2.73,0,0,0-2.52,2.12,10.62,10.62,0,0,0-.24,1.12c0,1.84-.12,3.72-.12,5.56a9.1,9.1,0,0,0,.4,2.32A2.48,2.48,0,0,0,67.64,101a11,11,0,0,0,2.8.08,3.15,3.15,0,0,0,3-3.44Z"/></g></svg>
```
Comment (gael-boyenval) [inline src/icons/48px/User_Payment_Colors_PayLib_48px.svg]: no colors icons

File: src/icons/48px/User_Payment_Colors_Paypal_48px.svg [hunk 9]
```diff
@@ -0,0 +1 @@
....5.77.77,0,0,0,0,.15l3.14,9.2-2.95,4.16a.49.49,0,0,0,.12.68.47.47,0,0,0,.28.09H90a.81.81,0,0,0,.67-.35l9.49-13.65a.5.5,0,0,0-.12-.69A.49.49,0,0,0,99.77,119.82Z"/><path class="cls-4" d="M130.18,114.77l-2.43,15.43a.51.51,0,0,0,.42.57h2.53a.78.78,0,0,0,.8-.69l2.4-15.16a.49.49,0,0,0-.41-.56h-2.8A.5.5,0,0,0,130.18,114.77Z"/><path class="cls-4" d="M115.75,72.7c0,.3-.1.6-.16.91-2,10.53-9.06,14.16-18,14.16H93a2.22,2.22,0,0,0-2.19,1.88l-2.33,14.8-.66,4.2A1.16,1.16,0,0,0,89,110h8.09A2,2,0,0,0,99,108.36l.08-.41,1.52-9.66.1-.53a2,2,0,0,1,1.92-1.65h1.21c7.83,0,14-3.18,15.76-12.39.75-3.84.36-7-1.62-9.31A7.73,7.73,0,0,0,115.75,72.7Z"/><path class="cls-6" d="M113.6,71.85c-.31-.09-.63-.18-1-.25s-.68-.14-1-.19a25.19,25.19,0,0,0-4-.3H95.41a1.94,1.94,0,0,0-1.92,1.65L90.9,89.17l-.08.48A2.22,2.22,0,0,1,93,87.77h4.56c9,0,16-3.63,18-14.16.06-.31.11-.61.16-.91a11.28,11.28,0,0,0-1.69-.71Z"/><path class="cls-5" d="M93.49,72.76a1.92,1.92,0,0,1,1.08-1.45,1.86,1.86,0,0,1,.84-.2h12.18a25.19,25.19,0,0,1,4,.3c.35,0,.69.12,1,.19l1,.25.46.14a11.28,11.28,0,0,1,1.69.71c.61-3.89,0-6.53-2.11-8.93-2.32-2.64-6.5-3.77-11.85-3.77H86.24a2.22,2.22,0,0,0-2.19,1.88l-6.47,41a1.33,1.33,0,0,0,1.31,1.54h9.6L90.9,89.17Z"/></g></svg>
```
Comment (gael-boyenval) [inline src/icons/48px/User_Payment_Colors_Paypal_48px.svg]: no colors icons

File: src/icons/48px/User_Payment_Colors_Visa_48px.svg [hunk 10]
```diff
@@ -0,0 +1 @@
...7H168m0-1H24a8,8,0,0,0-8,8v96a8,8,0,0,0,8,8H168a8,8,0,0,0,8-8V48a8,8,0,0,0-8-8Z"/><path class="cls-4" d="M168,40H24a8,8,0,0,0-8,8V64H176V48A8,8,0,0,0,168,40Z"/><path class="cls-5" d="M16,128v16a8,8,0,0,0,8,8H168a8,8,0,0,0,8-8V128Z"/><rect class="cls-6" x="16" y="40" width="160" height="112" rx="8"/><polygon class="cls-4" points="75.44 112.68 81.12 79.4 90.2 79.4 84.52 112.68 75.44 112.68"/><path class="cls-4" d="M117.4,80.24a23.17,23.17,0,0,0-8.12-1.4c-9,0-15.28,4.52-15.32,11,0,4.8,4.52,7.44,8,9s4.68,2.68,4.68,4.12c0,2.24-2.8,3.28-5.44,3.28a19,19,0,0,1-8.52-1.76L91.48,104,90.2,111.4a29,29,0,0,0,10.12,1.76c9.52,0,15.68-4.44,15.76-11.36,0-3.8-2.36-6.68-7.6-9-3.16-1.52-5.12-2.56-5.08-4.12s1.64-2.88,5.16-2.88a16.69,16.69,0,0,1,6.8,1.28l.8.4,1.24-7.2"/><path class="cls-4" d="M140.68,79.44h-7c-2.16,0-3.8.6-4.72,2.76l-13.48,30.48H125l1.92-5h11.6c.28,1.16,1.12,5,1.12,5H148L140.68,79.4m-11.2,21.44,3.6-9.28c0,.08.76-2,1.2-3.2l.64,2.88,2.08,9.6Z"/><path class="cls-4" d="M67.84,79.44,59,102.12,58,97.52c-1.68-5.32-6.8-11.08-12.56-14l8.12,29.08h9.6l14.24-33.2h-9.6"/><path class="cls-4" d="M50.72,79.4H36.12l-.12.72c11.36,2.72,18.92,9.4,22,17.4l-3.2-15.28c-.56-2.12-2.12-2.76-4.12-2.84"/></g></svg>
```
Comment (gael-boyenval) [inline src/icons/48px/User_Payment_Colors_Visa_48px.svg]: no colors icons

**Comment by melanieanglardLMFR** [comment]
I will add the `WIP` status to take time to clean that because i trust Antho for that and i didn't check the package ;)

**Comment by melanieanglardLMFR** [comment]
@gael-boyenval , i don't know what append but my branch is not deployed well. Do you know why ?

**Comment by gael-boyenval** [comment]
we still have : 

multiple empty icons that does not seem to match the category : 
Device_Empty_19_32px

- syntax error in icon names : Contol_TagClose (control instead of control)
- syntax error : Navigation_Notification_Avaible_48px (avaible instead of available)

display error on Publish_Tag--Delete

Product_Promise_Scan_48px => does not display properly

Stars full and alf : colors are not working.  =>
Social_View_Star--First / Social_View_Star--full / empty / alf = same thing to us

duplicated => Social_View_FirstView_48px

Store_RelayLocation--Filled_48px / Store_StoreLocation--Filled_48px => not flattened

Store_relayLocation--Filled_64px => not flattened + naming error (relay instead of Relay uppercase)

**Pull request closed without merging**